app/views/jst/courses/roster/rosterUser.handlebars

```diff
@@ -2,7 +2,11 @@
   {{>avatar}}
 </td>
 <td>
-  <a href="{{html_url}}" class="roster_user_name">{{name}}</a>
+  {{#if canViewSisId}}
+  <a href="{{html_url}}" class="roster_user_name">{{name}}</a> ({{short_name}})
```

Comment:
Also, in the interest of internationalization, this should now use something similar to this as well (when short_name isn't blank):
```
{{#t}}<a href="{{html_url}}" class="roster_user_name">{{name}}</a> ({{short_name}}){{/t}}
```